.github/workflows/rust.yml

```diff
@@ -16,28 +16,42 @@ env:
 jobs:
   build_clippy_fmt:
     runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        target: ["nanos", "nanox", "nanosplus"]
+
     steps:
       - name: arm-none-eabi-gcc
         uses: fiam/arm-none-eabi-gcc@v1.0.3
         with:
           release: '9-2019-q4'
       - uses: actions/checkout@v2
-      - name: Install thumbv6 target
+      - name: Install thumbv6 target (Nano S, Nano X)
         uses: actions-rs/toolchain@v1
         with:
           toolchain: nightly
           override: true
           target: thumbv6m-none-eabi 
```

Comment:
I believe `target` is not required anymore and can be removed. See https://github.com/aiooss-ledger/ledger-nano-piv/blob/main/.github/workflows/rust.yml#L27
This means that next step `Install thumbv8 target (Nano S Plus)` can be removed.